### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by imran5644**
Created a clone of Newsweek web page where I designed the whole website using bootstrap in :-
1- navbar
2- footer
3-sections of news-headline
4-images of the news related topics.
5- created vs class for the debate section.
6-created us class for u.s section to fix the image size
7- created bp class for bottom padding

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+- ![](https://img.shields.io/badge/HTML-red)
+
+## Live Demo
+
+[Live Demo Link](imran5644.github.io/Newsweek_bootstrap/)
+
+##  Getting Started
+
+To run the code simply download the repository by either cloning it or 
+downloading it as Zip file. Then use the old trustable browser to open the HTML file.
```
Comment (descholar-ceo) [inline README.md:28]: Try to elaborate more steps that a user needs to do after downloading or cloning this project.

File: stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+  border-bottom: 4px solid lightgray;
+  width: 330px;
+}
+
+.debate-img img {
+  width: 70px;
+  height: 70px;
+}
+
+.bp {
```
Comment (descholar-ceo) [inline stylesheets/style.css:84]: Try to make this class more descriptive

File: stylesheets/style.css [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+
+.digital {
+  font-size: 10px;
+}
+
+.footer-logo svg {
+  width: 180px;
+}
+
+.us img {
```
Comment (descholar-ceo) [inline stylesheets/style.css:235]: try to make this class `us` descriptive

File: stylesheets/style.css [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+
+.debate1 h2 {
+  width: 250px;
+}
+
+.debate1 p {
+  width: 250px;
+}
+
+.vs {
```
Comment (descholar-ceo) [inline stylesheets/style.css:282]: Make this class `vs` descriptive

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @imran5644  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: Your project is amazing so far, and it's almost looks like the original page
### :warning: However we need to make few changes to mark this project approved. I am sure by implementing all requested changes, you will end up with a very good clone. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] Live demo link doesn't work from readme file
- [x] Please work again on your Pull request description, and its title should reflect the overall work you have done on this Pull Request.
- [x] Main navigation links should be clickable links
- [x] Login link in navbar should be clickable
- [x] Login link should disappear on all screens below of width 768px
- [x] Screens of size 992 - 1028 px, the navbar navigation links become a mess

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-10-27 20-24-11 | Screenshot from 2020-10-27 20-26-56 |

:point_right: Could you see the difference :question: (Those screenshots are taken from a 998 px screen)

- [x] Make the subscribe button in navbar, looks like the original, by changing font-color, and font-size
- [x] Try to give some marin to the `In the magazine` and `Editor's pick` title, currently is touching the boxes on bottom, and make it looks like the original
- [x] Try to reduce space between all columns on big screens and medium screens
- [x] The debate section is breaking on medium screens, it should like the orinal one on medium screens

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-10-27 20-29-47|Screenshot from 2020-10-27 20-30-53|
:point_right: Could you see the difference :question: (Those screenshots are taken from a 998 px screen)

- [x] Pictures from in the magazine sections, are stretched, on all screens, try to make them responsive
- [x] Please remove the horizontal scrollbar on the bottom of your clone
Screenshot from 2020-10-27 20-31-58
- [x] Fix the newsletter section

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-10-27 20-33-14|Screenshot from 2020-10-27 20-34-03|
:point_right: Could you see the difference :question: (Those screenshots are taken from a 998 px screen)
- [x] The pictures from `Editors pick` are overlapping one another (on screens between 990 - 1025 px)
- [x] The footer is not well aligned on leftside (all screens)
- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [ ] You should add dummy weather data above of date info, to make it look professional
- [ ] Increase the size of Newsweek logo in navbar on big screens, and adjust it depending on the screen size
- [ ] The navbar should have a polygon on its right bottom ending

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-10-27 20-48-50 | Screenshot from 2020-10-27 20-49-45 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Your commit messages should be in imperative mode or present tense, you should have used `Update in the magazine section for tablet` instead of `updated in the magazine section for tablet`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: stylesheets/style.css (hunk 3 above)
Comment (imran5644) [reply stylesheets/style.css:235]: @descholar-ceo Thanks for the code review. I have removed the errors and updated the code.

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
### Hello @imran5644 :wave: 
#### You've done well in implementing some of the changes the previous TSE requested.
However, there are still a few changes that you need to make before the project can be approved.
### Required Changes :writing_hand: 
- [x] Add some margin to the columns in the editor's pick. Everything looks jampacked at screen size of about 1300px to 1026px.
Screenshot from 2020-10-27 23-36-10
- [x] Your subscribe section doesn't look like that of the original. Try to use the same color, fonts, and spacing as in the original.
- Your Clone
Screenshot from 2020-10-27 23-42-48
- What your clone looks like at about 1034px
Screenshot from 2020-10-27 23-50-40

- Original
Screenshot from 2020-10-27 23-44-27
- [x] Most of the issues are with the responsiveness of your page. Please take the time to make the different elements of the page respond well to different screen sizes just like the original website.
Once you are done making those changes, please request another review.

Happy coding :+1: 
slack(@ebeagusamuel)

**Review by Georjane — APPROVED** [review]
# Status: PR Approved ✔️

Hi @imran5644 
## Nice work implementing the required changes by the previous TSE!

You did a great job using Bootstrap and your mobile version looks really nice. 👏

You can go ahead and merge this PR and 🤔it will also be very nice if you add some description to your Readme file especially on the section on Getting Started. You can achieve this by adding the commands for cloning the repo.

Goodluck on your next project 👍

**Pull request merged**